polaris-react/src/components/Pagination/Pagination.scss

```diff
@@ -59,4 +59,43 @@
       /* stylelint-enable */
     }
   }
+
+  &.table {
+    button {
+      #{$se23} & {
```

Comment:
I already resolved this but it snuck back in 😳 just delete this whole block the style `&.table` is above!